tests/qchem/test_tapering.py

```diff
@@ -952,3 +958,25 @@ def test_inconsistent_callable_ops(operation, op_wires, op_gen, message_match):
         taper_operation(
             operation, generators, paulixops, paulix_sector, wire_order, op_wires, op_gen
         )
+
+
+@pytest.mark.parametrize(
+    ("ps_size", "max_size"),
+    [
+        (19, 49),
+        (4, 13),
```

Comment:
```suggestion
        (190, 49),
        (40, 13),
```

Let's make sure the pauli sentences are larger than the max size.  Otherwise, no splitting will actually occur.

Reply:
Thanks for catching this. Done!